apps/cookbook/src/app/examples/radio-example/examples/reactive-forms.ts

```diff
@@ -100,29 +82,29 @@ toggleRequired(required: boolean) {
   template: config.template,
   styles: [
     `
-      :host {
-        display: flex;
+      kirby-checkbox:not(:first-of-type) {
+        margin-top: 4px;
+      }
```

Comment:
Why is this needed? 🤔

Reply:
I tried to make the configuration checkboxes look like the one for modal example. I didn't succeed. It could be nice with some uniformity for our configurations, which is why I wanted to use example configuration wrapper with check boxes to make it similar to the modal configuration.